code/modules/events/shuttle_catastrophe.dm

```diff
@@ -5,7 +5,7 @@
 	max_occurrences = 1
 
 /datum/round_event_control/shuttle_catastrophe/canSpawnEvent(players, gamemode)
-	if((SSshuttle.emergency.name == "Build your own shuttle kit") || (SSshuttle.emergency.mode != SHUTTLE_CALL || SSshuttle.emergency.mode != SHUTTLE_RECALL || SSshuttle.emergency.mode != SHUTTLE_IDLE))
```

Comment:
if NOT in idle state return false

4head

Reply:
well then

Reply:
the jamie's code literally always returned false